index.bs

```diff
@@ -176,19 +185,101 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
+
+NOTE: When such a progressive decoding of the layers within an [=Operating Point=] is not desired or when a controlled rendering of the different output images is desired (e.g. grid of layered images), multiple [=AV1 Image Items=] sharing the same [=AV1 Image Item Data=] can be created and associated with different 'lsel' properties, each with a different value of 'layer_id'. In the case of an image grid composed of 4 layered images, each made of 2 layers, controlled rendering could be achieved by creating a set of items as follows: 4 items each corresponding to the low quality/resolution image as decoded from the first layer of each of the 4 layered images, i.e. with 'layer_id' set to 0; 4 items corresponding the second layer, i.e. with 'layer_id' set to 1; a grid item composing the first set of 4 images; another grid item composing the second set of 4 images.
+
+<h4 id ="layered-properties">Properties</h4>
+<h5 id ="operating-point-selector-property">Operating Point Selector Property</h5>
+
+<h6 id ="operating-point-selector-property-definition" class="no-toc">Definition</h6>
+
+<pre class="def">
+  Box Type:       <dfn value export for="OperatingPointSelectorProperty">a1op</dfn>
+  Property type:  Descriptive item property
+  Container:      <code>ItemPropertyContainerBox</code>
+  Mandatory:      No
+  Quantity:       One
+</pre>
+
+<h6 id="operating-point-selector-property-description" class="no-toc">Description</h6>
+
+An <dfn>OperatingPointSelectorProperty</dfn> may be associated with an [=AV1 Image Item=] to provide the index of the operating point to be processed for this item. If associated, it shall be marked as essential.
+
+In the absence of a 'lsel' property associated with the item:
+- If an [=OperatingPointSelectorProperty=] is associated with an [=AV1 Image Item=], the 'ispe' property associated with this item shall describe the dimension of the output image corresponding to the operating point.
+- If it is not associated with an item, the 'ispe' documents the dimensions of the the operating point whose index is 0.
+
+<h6 id ="operating-point-selector-property-syntax" class="no-toc">Syntax</h6>
+
+```
+class OperatingPointSelectorProperty extends ItemFullProperty('a1op', version = 0, flags) {
+    u8 op_index;
+}
+```
+
+<h6 id ="operating-point-selector-property-semantics" class="no-toc">Semantics</h6>
+
+<dfn noexport>op_index</dfn> indicates index of the operating point to be processed for this item. Its value shall be between 0 and [=operating_points_cnt_minus_1=].
+
+<h5 id ="layer-selector-property">Layer Selector Property</h5>
+The 'lsel' property defined in [[!HEIF]] may be associated with an [=AV1 Image Item=]. The 'layer_id' indicates the value of the spatial id to render. The corresponding spatial layer shall produce an output frame. Other layers may be needed to decode the indicated layer.
```

Comment:
Do we need to explicitly state that `layer_id` should be within a certain range or something like that (like how we do for `a1op` and `operating_points_cnt_minus_1`)?

It's probably not needed, but thought I'd check just to be sure.

Reply:
It would be a good clarification, yes.

Reply:
done.